extensions/azurecore/package.json

```diff
@@ -114,20 +114,6 @@
               "Verbose",
               "All"
             ]
-          },
-          "azure.authenticationLibrary": {
```

Comment:
@cheenamalhotra What about having a one-time toast popup for anyone who :

1. Has an Azure account
2. Has ADAL currently configured as their auth library

Reply:
Makes sense, I just fetched telemetry and noticed that we still have ADAL being used - very sparingly (<0.2%) but it's not yet 0. I wonder if this was done for testing reasons or an actual user. We can give it another release and keep this PR in draft until we get to 0.

Reply:
If you do that then I'd probably suggest having a popup on every launch telling users still using it that it'll be deprecated and removed in a later release. Otherwise there's probably people who don't even realize they have it enabled that will just always keep it on until forced.